style.css

```diff
@@ -473,21 +391,330 @@ textarea {
 
 .text-box {
   margin-bottom: 24px;
-  list-style: none;
 }
 
-.button-contact {
+.contact-form-2 {
+  display: flex;
+  flex-direction: column;
+  align-content: center;
+  align-items: center;
+  justify-content: space-between;
+  flex-wrap: wrap;
+  text-align: center;
+  background-image: url("final-pic.png");
+  background-repeat: no-repeat;
+  background-size: 60px;
+  background-position: right 60%;
+}
+
+.my-email {
+  font-family: "Inter", sans-serif;
+  font-weight: 400;
+  font-size: 20px;
+  text-align: center;
+  color: #172b4d;
+  margin: 0 0 67px 0;
+}
+
+.footer {
+  background-color: black;
+  width: 150px;
+  height: 4px;
+  border-radius: 24px;
+  margin-top: 100px;
+}
+
+.submit-b {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  letter-spacing: 0.03em;
+}
+
+.last-sec {
+  display: flex;
+  justify-content: center;
+}
+
+.last-sec p {
+  text-align: center;
+  width: 375px;
+  height: 189px;
+  margin-top: 10px;
+  padding: 0 0 0 0;
+}
+
+.icon-container-2 {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  margin: 46px 100px 0 100px;
+  padding: 0;
+}
+
+.icon-container-2 li {
+  padding: 0 10px 10px 10px;
+  align-items: center;
+  width: 100%;
+}
+
+.icon-container-3 {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  margin: 0 100px 0 100px;
+  padding: 0;
+}
+
+.icon-container-3 li {
+  padding: 0 10px 10px 10px;
+  align-items: center;
+}
+
+.about-me-3 {
+  margin-top: 40px;
+  padding: 6% 10% 6% 10%;
+}
+
+.project-card {
+  display: flex;
+  flex-direction: column;
+  background-color: #ebf0ee;
+  width: 328px;
+  height: 474px;
+  margin: 5px 0;
+  border: 1px solid #d0d9d4;
+  border-radius: 8px;
+}
+
+.project-title {
   display: flex;
+  font-family: "Inter", sans-serif;
+  font-style: normal;
   justify-content: center;
   align-items: center;
+  font-weight: 600;
+  font-size: 24px;
+  padding: 0% 10% 0% 10%;
+  color: #3a4a42;
+  line-height: 32px;
+  text-align: center;
+  background-color: white;
+  margin-top: 271px;
+}
+
+.project-abilites {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  margin: 0 0 0 0;
+  padding: 10px 0 10px 0;
+  background-color: white;
+}
+
+.b_div {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  background-color: white;
   margin: 0 0;
   border-radius: 8px;
+  padding: 10px 12px 10px 12px;
   gap: 10px;
-  font-size: 40px;
 }
 
-.last-icon {
-  border-radius: 24px;
-  margin-top: 30px;
-  margin-bottom: 40px;
+.see-button {
+  background-color: #36b37f;
+  font-weight: 600px;
+  font-size: 17px;
+  color: white;
+  border-radius: 4px;
+  width: 122px;
+  height: 48px;
+  border: 2px solid #36b37f;
+  transition: background-color 1s;
+}
+
+.see-button:active {
+  background: #008552;
+  border-radius: 4px;
+}
+
+.see-button:hover {
+  background-color: #d0d9d4;
+  box-shadow: 0 8px 16px rgba(54, 179, 127, 0.24);
+}
```

Comment:
Please consider adding the functionality of `hover and press` on the button. So when the button is hovered or pressed its style changes as follows. Currently, your styles are not in accordance with the design. To achieve this you can add the following functionality of code into your CSS file.

## Solution:--

```
.classname:hover{
    background-color: #36B37F;
    color: white;
}
```

```
.classname:active{
    background-color: #008552;
    color: white;
}
```

## Demonstration:--

![image](https://user-images.githubusercontent.com/94223367/222924036-ec7c1353-54de-484a-b446-68be07b7ae09.png)